.github/ISSUE_TEMPLATE/ask-question.yml

```diff
@@ -0,0 +1,50 @@
+name: 🙋‍♂️ Ask a question
+description: Tell us what's on your mind
+title: "[question]: "
+labels: ["triage"]
+# assignees:
+#   - assign your sdk team e.g., onesignal/unity
```

Comment:
This should be uncommented and changed to the team responsible for web SDKs. I don't know the teams, so you may have to ask @emawby.